eos/rare-b-decays/observables.cc

```diff
@@ -1111,9 +1150,9 @@ namespace eos
                 make_expression_observable("B->K^*ll::NormalizedBR", R"(\mathcal{B}(\bar{B}\to \bar{K}^*\ell^+\ell^-)/\mathcal{B}(\bar{B}\to \bar{K}^*J/\psi))",
                         Unit::None(),
                         R"(
-                        <<B->K^*ll::BR>>
+                        <<B->K^*ll::BR;model=WET>>
```

Comment:
I am a bit on the fence about this, but let's keep it this way for the time being.

Reply:
Yes we need to discuss this. I couldn't find an elegant solution for the moment.